server/activefiles.go

```diff
@@ -49,26 +49,26 @@ type currentUpload struct {
 	totalFileBytes int
 }
 
-func NewActiveFileManager(fileStore FileStore) *ActiveFileManager {
-	return &ActiveFileManager{
-		activeFiles: make(map[string]*ActiveFile),
+func newActiveFileManager(fileStore fileStore) *activeFileManager {
+	return &activeFileManager{
+		activeFiles: make(map[string]*activeFile),
 		fileStore:   fileStore,
 	}
 }
 
-func (self *ActiveFileManager) PrepareUpload(fileExtension string, userKey string) string {
-	self.Lock()
-	defer self.Unlock()
+func (activeFileManager *activeFileManager) PrepareUpload(fileExtension string, userKey string) string {
```

Comment:
Highly suggested to use a shorter name here, possibly `afm` or just `m` (for manager).

See https://github.com/golang/go/wiki/CodeReviewComments#receiver-names.